Update/ura_001.txt

```diff
@@ -1457,13 +1457,13 @@ void main()
 //r無意識のうちに剣呑な口調になってしまったのも、きっと私には言えない疲れや、悩みを抱えているからなのかもしれない。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "無意識のうちに剣呑な口調になってしまったのも、きっと私には言えない疲れや、悩みを抱えているからなのかもしれない。",
-			NULL, "Maybe I slipped into a nasty tone. I'm tired and can't really tell.", Line_Normal);
+			NULL, "I'm sure the reason she slipped into a nasty tone is because she's got things weighing on her mind that she can't tell me.", Line_Normal);
```

Comment:
```suggestion
			NULL, "I'm sure the reason she slipped into a nasty tone is because she has things weighing on her mind that she can't tell me.", Line_Normal);
```